apps/system/test/unit/utility_tray_test.js

```diff
@@ -769,6 +769,54 @@ suite('system/UtilityTray', function() {
     });
   });
 
+  suite('click and mouse events', function() {
+    var evt;
+
+    setup(function() {
+      evt = {
+        type: 'click',
+        target: UtilityTray.grippy,
+        preventDefault: this.sinon.spy()
+      };
+    });
+
+    test('it should check if preventDefault on mousedown', function() {
+      evt.type = 'mousedown';
+      UtilityTray.handleEvent(evt);
+      assert.isTrue(evt.preventDefault.called);
+    });
+
+    test('it should check if preventDefault on mousemove', function() {
+      evt.type = 'mousemove';
+      UtilityTray.handleEvent(evt);
+      assert.isTrue(evt.preventDefault.called);
+    });
+
+    test('it should check if preventDefault on mouseup', function() {
+      evt.type = 'mouseup';
+      UtilityTray.handleEvent(evt);
+      assert.isTrue(evt.preventDefault.called);
+    });
```

Comment:
Those first 3 tests are a bit confusing... I get the intent but still.

If we just have `mousedown` to handle, I'd like to have just the bottom 3 tests with a `mousedown` event + 1 test with another target where we `assert.isFalse(evt.preventDefault.called)`.

Also if you want the tests to be "more real" you can dispatch real events and check `evt.defaultPrevented` on them.